kaldi/install.bash

```diff
@@ -20,7 +20,7 @@ tue-install-system-now zlib1g-dev automake autoconf patch grep \
     gstreamer1.0-plugins-good gstreamer1.0-plugins-ugly
 
 # Numpy later than 1.15.0 is required
-pip install --user numpy>=1.15.0
+pip install --user "numpy>=1.15.0"
```

Comment:
Are the quotes around numpy the correct way? Because with no quotes an empty file named `=1.15.0` is created

Reply:
I think quotes are correct.

Reply:
Well then this is good to go